cmd/awscollector/main.go

```diff
@@ -46,8 +46,11 @@ func main() {
 	}
 
 	// init cfgFactory
-	cfgFactory := config.GetCfgFactory()
-	if cfgFactory == nil {
+	cfgFactory := config.GetParserProvider()
+	// TODO(pingleig): a hack to keep consistent behaviour with previous code
+	// so we exit 0 when running on vm for config error.
+	// https://github.com/aws-observability/aws-otel-collector/issues/340
```

Comment:
we should not put personal info in code/comment. :)

Reply:
It's actually old habit from https://google.github.io/styleguide/cppguide.html#TODO_Comments, the internal platform support searching by todo author and jump to author's photo page when you click the username.